tool/tsh/tsh.go

```diff
@@ -408,6 +408,7 @@ func Run(args []string, opts ...cliOption) error {
 	dbLogin.Arg("db", "Database to retrieve credentials for. Can be obtained from 'tsh db ls' output.").Required().StringVar(&cf.DatabaseService)
 	dbLogin.Flag("db-user", "Optional database user to configure as default.").StringVar(&cf.DatabaseUser)
 	dbLogin.Flag("db-name", "Optional database name to configure as default.").StringVar(&cf.DatabaseName)
+	dbLogin.Flag("cluster", clusterHelp).StringVar(&cf.SiteName)
```

Comment:
How about other `tsh db` subcommands? And while we're here, can we also make sure `tsh app` commands support it also?

Reply:
besides `list` and `login`, there are also `logout`/`env`/`config`/`config`. Can all these subcommands take in `--cluster`? i think that makes sense but i don't have multiple cluster to try (maybe i should setup one).

will add to `tsh app`

Reply:
I think we should add it to all for consistency. Yes, can you set it up, otherwise you're not gonna be able to make sure this works. You can setup a root and a leaf cluster on the same machine even. See trusted clusters docs: https://goteleport.com/docs/setup/admin/trustedclusters/.